projects/js-packages/charts/src/components/shared/test/use-element-height.test.tsx

```diff
@@ -0,0 +1,278 @@
+import { renderHook, waitFor } from '@testing-library/react';
+import { useElementHeight } from '../use-element-height';
+
+// Mock ResizeObserver
+class MockResizeObserver {
+	callback: () => void;
+	entries: ResizeObserverEntry[];
+	element: HTMLElement | null;
+
+	constructor( callback: () => void ) {
+		this.callback = callback;
+		this.entries = [];
+	}
+
+	observe( element: HTMLElement ) {
+		this.element = element;
+		// Trigger initial callback
+		this.callback();
+	}
+
+	disconnect() {
+		this.element = null;
+	}
+
+	// Helper method to simulate resize
+	triggerResize() {
```

Comment:
This does not seem to be used.

Reply:
Removed in 6ecce387040e8f8cff365f19acd9228b8dd9a943. Thanks!